Source/UI/Scripts/d3/nemeStatsPointsLineGraph.js

```diff
@@ -0,0 +1,29 @@
+﻿var playerData = [];
```

Comment:
Global variables and global functions are something that we are trying to avoid. This needs to be refactored.